doc/api/wasi.md

```diff
@@ -0,0 +1,86 @@
+# WebAssembly System Interface (WASI)
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+The WASI API provides an implementation of the [WebAssembly System Interface][]
+specification. WASI gives sandboxed WebAssembly applications access to the
+underlying operating system via a collection of POSIX-like functions.
+
+```js
```

Comment:
Can we give this example a file name and then include a line with `node --experimental-wasi example.js`?

Alternatively a comment at the top of the example, or a note above or below to indicate Node.js should be run with the `--experimental-wasi` flag.

Reply:
We should probably also document that `--experimental-wasm-bigint` is likely needed as well.